scormxblock/scormxblock.py

```diff
@@ -28,7 +29,7 @@
 
 SCORM_ROOT = os.path.join(settings.MEDIA_ROOT, "scormxblockmedia")
 SCORM_URL = os.path.join(settings.MEDIA_URL, "scormxblockmedia")
-MAX_WORKERS = 10
+MAX_WORKERS = getattr(settings, "THREADPOOLEXECUTOR_MAX_WORKERS", 10)
```

Comment:
Should we get it from settings? or directly from ENV_TOKENS?
Actually, I just read that comment from the implementation of philU
[philU scorm](https://github.com/philanthropy-u/openedx-scorm-xblock/blob/master/openedxscorm/scormxblock.py#L33)